t2.py

```diff
@@ -0,0 +1,51 @@
+
+def weak_encrypt(data):
+    key = "static_key"  
+    encrypted = "".join(chr(ord(c) ^ ord(k)) for c, k in zip(data, key))
+    return encrypted
+
+DB_USERNAME = "admin"  
+DB_PASSWORD = "password123"  
+API_KEY = "sk_test_supersecretapikey12345"  
+
+PRIVATE_KEY = """
+-----BEGIN PRIVATE KEY-----
+MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAMJh2yfh3Dey8c+k
+Exbadcodefakeprivatekeydata++++++++++++++++++
+-----END PRIVATE KEY-----
+"""
+
+def connect_to_database():
+    connection_string = f"postgresql://{DB_USERNAME}:{DB_PASSWORD}@localhost/mydb"
+    print(f"Connecting to the database with: {connection_string}")  
```

Comment:
Hardcoding connection strings and printing them, especially with credentials, can lead to security risks by exposing sensitive information in logs. Use environment variables for credentials and avoid logging them.

**Label**: security

**Code suggestion**

```python
connection_string = f"postgresql://:{os.getenv('DB_PASSWORD')}@localhost/mydb"
# Avoid printing the connection string with credentials
```